test/dynamo/test_generator.py

```diff
@@ -420,6 +460,19 @@ def fn(t):
         with self.assertRaises(StopIteration):
             next(gen)
 
+    @unittest.expectedFailure
+    def test_reconstruct_generator_mutate_tensor(self):
```

Comment:
Can you file an issue for this once this PR goes in so that we can track it as a known issue?

Reply:
Here https://github.com/pytorch/pytorch/issues/146628